lib/doorkeeper/openid_connect/helpers/controller.rb

```diff
@@ -8,6 +8,7 @@ def authenticate_resource_owner!
           super.tap do |owner|
             next unless controller_path == Doorkeeper::Rails::Routes.mapping[:authorizations][:controllers] &&
               action_name == 'new'
+            raise Errors::OpenidConnectError unless pre_auth.valid?
```

Comment:
This will still fail on line 37 if we're missing a `redirect_uri` in our error_response